### ISSUE TITLE
zOS compiler problem with escaped characters in native module code

### ISSUE THREAD

**Issue by rnchamberlain** [ibmruntimes/node#97]
This code (in the appmetrics native module) is failing to compile:

`strScript = replaceInString("\\","/",strScript);`

Compiler output is:
```
../src/plugins/node/prof/nodeprofplugin_temp.cpp:149:47: warning: missing terminating '"' character [-Winvalid-pp-token]
                        strScript = replaceInString("\x5c\x22\x2c"/",strScript);
                                                                   ^
../src/plugins/node/prof/nodeprofplugin_temp.cpp:149:47: error: expected expression

```
0x5c, 0x22 and 0x2c are ASCII for backslash, double quote, comma

**Comment by anisha-rohra** [comment]
Working on fixing this issue! A known conversion issue related to escapes, will update you as soon as I'm done.

**Comment by rnchamberlain** [comment]
Note: this seems to be fixed, or has gone away, when I moved up from node v6.11.2 to latest v6.11.4 build on zOS. No change to the appmetrics C++ code, though there may have been other build changes.

**Comment by mmallick-ca** [comment]
@rnchamberlain this was a known issue that we did not quite know how to deal with it earlier. I have just committed a change which is propagating through our build and test.
https://github.com/ibmruntimes/node/commit/039aa4efa2b1dfdd072fb089bf8ba0a938bb7f61
If all goes well it should propagate and be available in the next beta(and possibly the z/OS trial driver). I am not sure how the problem disappeared with the upgrade to v6.11.4. I suspect that this is a false positive.
